pandas/core/strings.py

```diff
@@ -624,6 +624,7 @@ def str_pad(arr, width, side='left', fillchar=' '):
 
 def str_split(arr, pat=None, n=None, return_type='series'):
     """
+    Deprecated: return_types 'series', 'index', 'frame' are now deprecated 
```

Comment:
To explain: the first sentence is used in summary tables on the API pages, so this is not what we want to show there.